### ISSUE TITLE
merge into world_war_ii_global

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-maps/world_war_ii_g40_balanced#11]
https://github.com/triplea-maps/world_war_ii_global/pull/8

XMLs seem to be migrated now, need to move issues from this map to that repo, remove the listing from the download description, and then delete this repo.

**Comment by simon33-2** [comment]
Yes, correct. This repo can be removed. Well, once we update the .yaml file.

**Comment by panther2** [comment]
Issues #7 and #8 are already listed at the wwii-global repo, as they have always applied to all maps included.
